_translator-files/po/de/Installation-for-Android.po

```diff
@@ -1,20 +1,19 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
+# ann0see <20726856+ann0see@users.noreply.github.com>, 2022.
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2022-08-16 16:56+0000\n"
-"Last-Translator: ann0see <please@seeGitHubOrWeblate.local>\n"
+"PO-Revision-Date: 2022-12-18 17:48+0000\n"
+"Last-Translator: ann0see <20726856+ann0see@users.noreply.github.com>\n"
```

Comment:
The change from upstream works ;-). Probably the next release is the last release where we need to remove the e-mails manually.